antea/reco/reco_functions.py

```diff
@@ -346,8 +346,8 @@ def find_coincidence_timestamps(tof_response: pd.DataFrame,
     Finds the first time and the IDs of the sensors used to calculate it
     for each one of two sets of sensors, given a sensor response dataframe.
     """
-    min1, time1 = find_first_time_of_sensors(tof_response, -sns1, n_pe)
```

Comment:
The test keeps working as it is, because the `find_first_time_of_sensors` function has not changed. What we miss is a test of `find_coincidence_timestamps`, but it's such a simple function that i don't know if it's worth it.

Reply:
On the other hand, the `sipm_shaping_convolution` has not changed: why do you want to change the test?

Reply:
Because the new files that such function will analyze are different due to the change in tof table. The test uses `tof_bin_size` and `evt_tof.time_bin` and it can be confusing for the user.

Reply:
I see what you mean: we could add a test with the new format, but I wouldn't remove the old one, because the function to be tested can work with both format and we want to continue using it for a while with the old format, too.

Reply:
Ok, perfect!

Reply:
Will you add the test files, then? I think you can push to this PR's branch, as well.

Reply:
Ok!